app/misc/authenticate.ts

```diff
@@ -139,9 +138,13 @@ function getUserInfo(callback) {
           $("#otpModal").modal('show');
         });
       }
-      else if (err.errno == "ENOTFOUND" || err.errno =="ENOENT"){
+      else if (401 == err.statusCode) {
```

Comment:
Is there a reasoning as to why err.statusCode is used here instead of err.errno, which is the error checking used for the Auth case below?

Reply:
Unfortunately, it's an entirely different object (HttpError), so the actual error is stored in a different variable name, so this is all we can do!

Reply:
Oh I see - makes sense.